test/run-tests

```diff
@@ -601,21 +573,21 @@ def check_commit_needs_testing(status, commands):
         return True
 
     # or the status is pending without a hostname in the description
-    if status["state"] == "pending" and not status.get("description"):
+    if status.state == "pending" and not status.description:
         logging.debug(
             "PR will be tested because it is in state 'pending' and has no description"
         )
         return True
 
     # or a generic re-check was requested:
-    if status["updated_at"] < commands[COMMENT_CMD_TEST_ALL]:
+    if status.updated_at.strftime("%Y%m%d-%H%M%S") < commands[COMMENT_CMD_TEST_ALL]:
```

Comment:
The gh api converts `status.updated_at` to DateTime object?

Reply:
I suggest converting this to a string once e.g.
```python
updated_at = status.updated_at.strftime("%Y%m%d-%H%M%S")
if updated_at < commands[COMMENT_CMD_TEST_ALL]:
...
```

Reply:
> The gh api returns the `status.updated_at` as a DateTime object?

Yes.
```
 |  updated_at
 |      :type: datetime.datetime
```